### DISCUSSION TITLE
The official `vips` package in Alpine Linux doesn't include AVIF support

### DISCUSSION THREAD

**Discussion by xamir82** [libvips/libvips#3495] [General]
I'm trying to write a Dockerfile for my app that uses libvips under the hood. The base image is an Alpine image and I'm using `apk` to install libvips in the image ([`vips-tools`](https://pkgs.alpinelinux.org/package/edge/community/armhf/vips-tools) in particular — e.g. `apk add vips-tools`).

The problem is the image doesn't come with AVIF support, `libheif` isn't listed as one of the dependencies of [`vips`](https://pkgs.alpinelinux.org/package/edge/community/armhf/vips). This is different, for example, from the Homebrew version, which does come with `libheif` by default.

Wouldn't it make sense to update the Alpine package to make it consistent with Homebrew in this regard? Especially as AVIF becomes increasingly more widely-adopted?

If not, could you please give me a hint or two about how I can add AVIF support to vips in the context of an Alpine Docker image?
I've tried installing [`libheif`](https://pkgs.alpinelinux.org/package/edge/community/armhf/libheif) or [`libheif-dev`](https://pkgs.alpinelinux.org/package/edge/community/armhf/libheif-dev) alongside `vips` but it doesn't seem to actually work.
Do I have no choice but to build from source?

Thanks.

**Comment by lovell** [comment]
Did you see https://pkgs.alpinelinux.org/package/v3.18/community/x86_64/vips-heif ?

**Reply by xamir82** [reply]
Oh, my bad, thank you.
